### ISSUE TITLE
Procurement and Support Strategies

### ISSUE THREAD

**Issue by Cribstone** [mainehackerclub/open_vehicle_tracker#12]
One of my contacts in urban planning forwarded me a document that outlines perfectly the kind of challenges this project aims to address. In bureaucrat language, the "RFP" which means "Request for Proposals", is the procedure in which all public institutions must follow when hiring a private sector contractor.  Apparently, the State of Maryland just released an RFP soliciting bids for a "Unified Bus System Architecture", for their fleet of passenger busses.  A central component of this architecture is a specification for an "Automatic Vehicle Location" (AKA AVL in bureaucrat/Vehicle Tracker in English) system.
   This document can be a critical reference for our own project as it outlines exactly the capabilities that major cities are looking for.  More significant to our current needs, the 500 page document shows exactly why the procurement process needs to be reformed.  Clearly there's no way an outfit like ours could ever actually bid on such a project, even if our solution met all the specifications and our bid came in significantly lower.  Simply making it through the 500 page RFP would require a dedicated team of specialists, which is exactly what most major government contractors devote their resources to...
The RFP can be found through this link- https://docs.google.com/file/d/0B5CZhhbWFEwPdVZ3Y3loVVlJVU0/edit

**Comment by janewilkin** [comment]
I can't say for certain, but I have a feeling that there are existing organizations in the state of Maine that would be interested in helping us with the RFP process.  Perhaps EMDC would be one of them.

**Comment by Cribstone** [comment]
I might venture to suggest our role should be to help them REFORM the RFP process rather than try to navigate it in its current state.  This project is a proof of concept for why the current system is inadequate.  Now if I could just get that golden email I've been waiting for we can meet with the RFP-EZ while we're down there.

**Comment by bschrader** [comment]
Yes, this spec was written for general contractors who will oversee an entire project implementation.  I don't think we want to become general contractors.

However, GCs need people like us.  There are opportunities in projects of this scale to provide our services and support as subcontractors, consultants and SaaS providers.

Good find.

On Tue, 09 Jul 2013 08:48:21 -0700
Andrew Jawitz notifications@github.com wrote:

>    One of my contacts in urban planning forwarded me a document that outlines perfectly the kind of challenges this project aims to address. In bureaucrat language, the "RFP" which means "Request for Proposals", is the procedure in which all public institutions must follow when hiring a private sector contractor.  Apparently, the State of Maryland just released an RFP soliciting bids for a "Unified Bus System Architecture", for their fleet of passenger busses.  A central component of this architecture is a specification for an "Automatic Vehicle Location" (AKA AVL in bureaucrat/Vehicle Tracker in English) system.
>    This document can be a critical reference for our own project as it outlines exactly the capabilities that major cities are looking for.  More significant to our current needs, the 500 page document shows exactly why the procurement process needs to be reformed.  Clearly there's no way an outfit like ours could ever actually bid on such a project, even if our solution met all the specifications and our bid came in significantly lower.  Simply making it through the 500 page RFP would require a dedicated team of specialists, which is exactly what most major government contractors devote their resources to...
> 
> ```
> The RFP can be found through this link- https://docs.google.com/file/d/0B5CZhhbWFEwPdVZ3Y3loVVlJVU0/edit
> ```
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/mainehackerclub/open_vehicle_tracker/issues/12

## 

Benjamin Schrader ben@unepic.com

**Comment by Cribstone** [comment]
We definitely don't want to become general contractors!  I included the spec on here as an example of the kind of inefficient procurement procedures that are ripe for disruption. I put it on here to show the extent to which conventional RFPs make it impossible for operations like ours to ever take part.  Its not a joke to say it requires a dedicated professional just to fill out the form, as I have literally seen large scale engineering firms devote 5 staffers to winning RFPs for every one who actually fulfills the contract once its awarded. 
    To the extent that we can pursue the OVT as a private operation, our business model should be based on the fact we can offer a feasible alternative to those already offered by conventional fleet management contractors who charge insanely high prices for government bids.  Part of the reason they charge so much is because they have to hire a team of layers just to fill the thing out.  Civic Startups will be able to compete for bids if reforms like RFP-EZ even the playing field by making applications more accessible AND if the cost difference between proprietary licensing and open source give Civic Startups enough of a competitive advantage to submit lower bids.
    So basically actively pushing to reform public procurement isn't just an activism issue.  Our business model depends on it!

**Comment by Cribstone** [comment]
Here is something which can potentially be a huge resource in the near future- A Civic Startup by the name of www.measuredvoice.com just made their "Government-Friendly Terms of Service free to be copied under a Creative Commons License"- http://measuredvoice.com/terms/  
  An explanatory blog post is here- http://blog.measuredvoice.com/post/54034472250/serving-the-servants
